project.clj

```diff
@@ -58,8 +63,14 @@
                                     :password :env/clojars_password
                                     :sign-releases false}]]
 
-  :profiles {:provided {:dependencies [[org.clojure/clojure "1.10.1"]
-                                       [org.clojure/clojurescript "1.10.520" :scope "provided"]]
+  :profiles {:provided {:dependencies [[org.clojure/clojure "1.10.3"]
+                                       [org.clojure/clojurescript "1.10.866" :scope "provided"]
+                                       [com.cognitect/transit-clj "1.0.324"]
```

Comment:
What are all those extra provided deps for?

Reply:
These are the latest transitive deps from clojurescript, satisfying  `:pedantic?`

Reply:
Got it. Might be a good idea to add some note about this, as I'm likely to forget otherwise. I never used `:pedantic?` myself.